CHANGES.md

```diff
@@ -1,3 +1,6 @@
+- [IMPROVED] Improved performance by reducing the number of HTTP requests made
+  when instantiating `Database` objects
```

Comment:
I would be specific here. This is a bit woolly. "Remove unneeded GET request to the database when creating `Database` objects with the create option set to `false`. It offered little protection as database could be deleted, credentials changed during the use of the object."